src/IssueViz/IssueVizPackage.cs

```diff
@@ -37,6 +39,7 @@ namespace SonarLint.VisualStudio.IssueVisualization
     [ProvideMenuResource("Menus.ctmenu", 1)]
     [ProvideToolWindow(typeof(IssueVisualizationToolWindow), MultiInstances = false, Style = VsDockStyle.Tabbed, Window = ToolWindowGuids.SolutionExplorer, Width = 325, Height = 400)]
     [ProvideToolWindowVisibility(typeof(IssueVisualizationToolWindow), Constants.UIContextGuid)]
+    [ProvideToolWindow(typeof(HotspotsToolWindow), MultiInstances = false, Style = VsDockStyle.Tabbed, Window = ToolWindowGuids.SolutionExplorer, Width = 325, Height = 400)]
```

Comment:
I think adding the commands to this package is fine.
We should have a chat about whether we want to create a separate package for the http listener package or not.